src/extension/package.json

```diff
@@ -74,6 +74,7 @@
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "npm run compile && node ./node_modules/vscode/bin/test",
```

Comment:
If we aren't using this `npm test` for anything, maybe you could replace it with your tests

```
"test": "yarn generate &&  yarn run-tests"
```

Reply:
I feel like test could still be useful if we add extension side tests. Not sure though.

Reply:
I think we should leave it in case we add other tests later. However, I did make this change so you only have to run one command to run both scripts:
    "template-tests": "node ./out/scripts/generate-test && node ./out/scripts/run-tests",
and removed "generate" and "run-tests"